easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'Bundle'
+
+name = 'Python-bundle'
+version = '3.10.4'
+
+homepage = 'https://easybuild.io/'
+description = """Python distribution with a number of widely used 
+extensions incl. NumPy, SciPy, Matplotlib, JupyterLab, MPI4PY, ... """
+
+local_gccver = '11.3.0'
```

Comment:
not needed, sub-toolchains are automatically mapped
```suggestion
```

Reply:
I hope to have some time to look at this tomorrow afternoon.

Reply:
Suggestion implemented.